Common/Securities/Option/OptionPriceModels.cs

```diff
@@ -110,11 +114,12 @@ public static IOptionPriceModel Integral()
             return new QLOptionPriceModel(process => new IntegralEngine(process),
                                            _underlyingVolEstimator,
                                            _riskFreeRateEstimator,
-                                           _dividendYieldEstimator);
+                                           _dividendYieldEstimator,
+                                           new[] { OptionStyle.European });
         }
 
         /// <summary>
-        /// Pricing engine for European options using finite-differences. 
+        /// Pricing engine for European options using finite-differences.
```

Comment:
Could you please update the method description/summary too? `CrankNicolsonFD` support both `American` and `European` option according to the `allowedOptionStyles` 15 lines below. I think we should add this information "for European", "for American", "for both American and European" in all descriptions/summary.